packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -295,6 +295,7 @@ class NavigationDestination extends StatelessWidget {
     this.selectedIcon,
     required this.label,
     this.tooltip,
+    this.disabled = false,
```

Comment:
You should be using a positive argument here. So `enabled` instead of `disabled`.

See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-double-negatives-in-apis

Reply:
Done